packages/corvid-local-logger/src/ParseError.js

```diff
@@ -0,0 +1,12 @@
+const ExtendableError = require("es6-error");
+
+class ParseError extends ExtendableError {
```

Comment:
This shouldn't be part of corvid-local-logger

Reply:
i can put it in utils

Reply:
done